web/test/factories.ts

```diff
@@ -183,10 +183,6 @@ export const generateSidebarCardContent = (overrides: Partial<SidebarCardContent
     ctaText: `some-cta-${randomInt()}`,
     notStartedHeader: `some-not-started-header-${randomInt()}`,
     completedHeader: `some-completed-header-${randomInt()}`,
-    imgPath: `some-img-path-${randomInt()}`,
-    color: `some-color-${randomInt()}`,
-    headerBackgroundColor: `some-header-background-color-${randomInt()}`,
-    borderColor: `some-border-color-${randomInt()}`,
     hasCloseButton: !!(randomInt() % 2),
```

Comment:
pebble, add `preBodySpanButtonText` to the generator function

Reply:
done